examples/timeseries/airQualityObservations.ttl

```diff
@@ -0,0 +1,47 @@
+@prefix tree: <https://w3id.org/tree#>.
+@prefix hydra: <http://www.w3.org/ns/hydra/core#>.
+@prefix dcterms: <http://purl.org/dc/terms/>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix lc: <http://semweb.mmlab.be/ns/linkedconnections#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix sosa: <http://www.w3.org/ns/sosa/>.
+@prefix tiles: <https://w3id.org/tree#> .
+@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix cot: <https://w3id.org/city_of_things#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+
+<https://mycity.example.org/airQualityObservations> a hydra:Collection ;
+                                                  rdfs:label "All airquality observations of our city."@en ;
+                                                  # Point to high-level nodes (per tile)
+                                                  void:subset </14/8393/5467> ; # This will redirect to the page with the latest sensor data from that tile, all observations within one tile can be found by following the "LowerThanRelation" recursively
+                                                  hydra:search _:b0 .
+
+_:b0  a  hydra:IriTemplate ;
+      hydra:variableRepresentation hydra:BasicRepresentation ;
+      hydra:template "https://lodi.ilabt.imec.be/air/{z}/{x}/{y}{?page}";
+      hydra:mapping [
+        a hydra:IriTemplateMapping ;
+        hydra:variable "x" ;
+        tree:path (sosa:hasFeatureOfInterest geo:longitude)  ;
+        hydra:property tiles:longitudeTile ;
+        hydra:required "true"^^xsd:boolean ;
+        sh:minInclusive "8393" ;
```

Comment:
This is really interesting! Didn’t think of this yet and it is not yet in the spec, but we could include this indeed!

Reply:
Can you make an issue to add this to the spec?